src/pages/Landing/NetworkStatus/index.tsx

```diff
@@ -147,20 +136,21 @@ export const NetworkStatus: React.FC<{ amountEth?: number }> = ({
       <Content isMobile={m}>
         <ScrollAnimation delay={750} animateIn="fadeIn" animateOnce>
           <Heading level={2} size="medium" color="blueDark" margin="none">
-            Network status
+            Total {TICKER_NAME} staked
           </Heading>
           <Text size="x-large" className="mt20">
             <BoldGreen className="mr10" fontSize={24}>
               {numberWithCommas(amountEth)} {TICKER_NAME}
             </BoldGreen>
-            already staked and counting.
           </Text>
           {thresholdReached ? (
             <>
               <Text className="mt20">
-                The threshold to launch the eth2
-                {IS_MAINNET ? ` mainnet ` : ` ${ETH2_NETWORK_NAME} testnet `}
-                has been reached 🎉
+                {IS_MAINNET
+                  ? `The Beacon Chain `
+                  : ` ${ETH2_NETWORK_NAME} testnet `}{' '}
+                needs at least {numberWithCommas(ETH_REQUIREMENT)} {TICKER_NAME}{' '}
+                staked to be secure.
```

Comment:
@ryancreatescopy Overall I'd argue against this even being included anymore. If I understand correctly, the beacon chain will no longer achieve finality if the number of ETH staked goes below 524,288 (vs still operating insecurely as this kind of implies), but at this point it seems more of a technical nuance, and something people can read the docs to learn more about. 

Now, I would argue the total ETH staked is mostly all that matters, in relation to the reward they can expect when financially choosing whether or not to commit their ETH to staking. It is nice to see visually that we're not at the very beginning anymore, and I believe the chart does this very nicely, but showing where we are now, and leave a large gap on the chart for where we expect it to go.